tests/jvm/src/test/resources/lowlevel.expect

```diff
@@ -345,9 +359,344 @@ Occurrences:
+  [0:0..0:1): * => _star_.
+  [0:2..0:12): unapplySeq => _root_.scala.collection.generic.SeqFactory#unapplySeq(CC).
+  [0:13..0:20): Nothing => _root_.scala.Nothing#
+[22:23..22:23):  => *.unapplySeq[Nothing]
+  [0:0..0:1): * => _star_.
+  [0:2..0:12): unapplySeq => _root_.scala.collection.generic.SeqFactory#unapplySeq(CC).
+  [0:13..0:20): Nothing => _root_.scala.Nothing#
+
+semanticdb/integration/src/main/scala/example/Methods.scala
+-----------------------------------------------------------
+
+Summary:
+Schema => SemanticDB v3
+Uri => semanticdb/integration/src/main/scala/example/Methods.scala
+Text => non-empty
+Language => Scala212
+Symbols => 66 entries
+Occurrences => 109 entries
+Diagnostics => 2 entries
+
+Symbols:
+_root_.example. => package example
+_root_.example.Example. => final object Example
+_root_.example.Methods# => class Methods.{+25 decls}
+  extends AnyRef
+_root_.example.Methods#$anon#x. => abstract val x: : Int
+  Int => _root_.scala.Int#
+_root_.example.Methods#AList# => type AList: [T >: Nothing <: Any] => Methods[T].this.List[T]
+  T => _root_.example.Methods#AList#[T]
+  Methods => _root_.example.Methods#
+  T => _root_.example.Methods#[T]
+  List => _root_.example.Methods#List#
+_root_.example.Methods#AList#[T] => typeparam T: >: Nothing <: Any
+  Nothing => _root_.scala.Nothing#
+  Any => _root_.scala.Any#
+_root_.example.Methods#List# => class List.{+1 decls}
+  extends AnyRef
+_root_.example.Methods#List#[T] => typeparam T: >: Nothing <: Any
+  Nothing => _root_.scala.Nothing#
+  Any => _root_.scala.Any#
+_root_.example.Methods#List#`<init>`(). => primaryctor <init>: (): Methods[T].this.List[T]
+  Methods => _root_.example.Methods#
+  T => _root_.example.Methods#[T]
+  List => _root_.example.Methods#List#
+  T => _root_.example.Methods#List#[T]
+_root_.example.Methods#[T] => typeparam T: >: Nothing <: Any
+  Nothing => _root_.scala.Nothing#
+  Any => _root_.scala.Any#
+_root_.example.Methods#`<init>`(). => primaryctor <init>: (): Methods[T]
+  Methods => _root_.example.Methods#
+  T => _root_.example.Methods#[T]
+_root_.example.Methods#`m8().`(). => def m8().: (): Nothing
+  Nothing => _root_.scala.Nothing#
+_root_.example.Methods#`m9().`# => class m9()..{+1 decls}
+  extends AnyRef
+_root_.example.Methods#`m9().`#`<init>`(). => primaryctor <init>: (): Methods[T].this.m9().
+  Methods => _root_.example.Methods#
+  T => _root_.example.Methods#[T]
+  m9(). => _root_.example.Methods#`m9().`#
+_root_.example.Methods#m1(). => def m1: : Nothing
+  Nothing => _root_.scala.Nothing#
+_root_.example.Methods#m10(AList). => def m10: (x: Methods[T].this.AList[T]): Nothing
+  x => _root_.example.Methods#m10(AList).(x)
+  Nothing => _root_.scala.Nothing#
+_root_.example.Methods#m10(AList).(x) => param x: Methods[T].this.AList[T]
+  Methods => _root_.example.Methods#
+  T => _root_.example.Methods#[T]
+  AList => _root_.example.Methods#AList#
+_root_.example.Methods#m11a(.type). => def m11a: (x: Predef.type): Nothing
```

Comment:
Missing test for disambiguating singleton types

```
class Foo[T]  {
  val x = ""
  object n { val x = List(1) }
  def m6(a: x.type) = ???
  def m6(b: n.x.type) = ???
}
```

and also test at reference site.

Reply:
Done.